requirements.txt

```diff
@@ -0,0 +1,36 @@
+astroid==2.8.4
```

Comment:
Do you actually need that many requirements?...

Reply:
Yes, you're right, some of them are redundant as these are all packages used, while some of them could have been automatically retrieved by pip dependency solver. This was fixed in https://github.com/MariaChizhova/SE_2021/pull/2 by using poetry instead of pip